cli/lib/build.js

```diff
@@ -95,6 +96,17 @@ const clearPublic = async () => {
   await execSync(`rm -rf ${publicDir}/*`, { cwd: rootDir });
 }
 
+const checkModuleValid = ()=> {
+  const isAllValid = moduleList.every(item=>{
+    return item.moduleName && item.moduleDir && fs.existsSync(item.moduleDir);
+  });
+
+  if (!isAllValid) {
+    logError('please check every moduleDir is exist in .env, otherwise, run "erda setup <module> <port>" to auto generate moduleDir');
```

Comment:
You can show user which moduleDir does not exist, make it easy for them to modify.